modules/ROOT/pages/config-add-proxy-whitelist.adoc

```diff
@@ -27,3 +27,53 @@ The following example defines regular expressions that allow requests to be made
 ----
 kubectl delete pod -n api-manager -l component=api-platform-web
 ----
+
+NOTE: This method is currently working from version 3.2.0 to version 3.2.1. From version 3.2.2 and any future versions, refer to Trusted Domains.
```

Comment:
```suggestion
For versions 3.2.2 and later, refer to Trusted Domains.
```

Reply:
Does "This method" refer to the last step or to the whole procedure? It would be better to have this information before the procedure.